NSS-Day-Cohort-66/rare-client-rare-poultry
# Jrm createpostclient1

# Create Posts
Users can now create posts

## To Tests

- ```git fetch``` branch ```jrm-createpostclient1```
- ```npm run dev```
- Click on Create Post in navbar
- Fill in the input fields and hit Publish button
- You should be directed to the Posts page where your new post should be displayed
- Check your database for your new post
